sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Blob/OciBlobDescriptor.cs

```diff
@@ -28,6 +29,8 @@ public partial class OciBlobDescriptor
 
         internal static string ComputeDigest(Stream stream)
         {
+            Debug.Assert(stream.CanSeek, "Should only be called on seekable streams.");
```

Comment:
Similar comment here - could we just take a MemoryStream instead?

Reply:
For this one, I looked at that and decided to handle it differently for this case, but I changed it to test CanSeek instead of the stream type.  Although I reworked that again this morning - let me check whether your suggestion works now in light of that refactoring ...

Reply:
I still have a dependency on this being Stream from the Download case ... would you be ok with me refactoring this when I address that?  I'll add a note to the issue, I'm tracking a few things there: https://github.com/Azure/azure-sdk-for-net/issues/32414

Reply:
Sure!